src/utils/graphqlClient.js

```diff
@@ -1,10 +1,28 @@
-import { ApolloClient, InMemoryCache } from '@apollo/client';
+import {
+  ApolloClient, InMemoryCache, ApolloLink, HttpLink,
+} from '@apollo/client';
 import env from './env';
 
 const BACKEND = env.REACT_APP_BACKEND_API;
+const AUTH = `${env.REACT_APP_AUTH_API}/api/auth/graphql`;
+const AUTH_FORCE_DEV = false;
+const AUTH_DEV = 'https://bento-dev.bento-tools.org/api/auth/graphql';
+
+const backendService = new HttpLink({
+  uri: BACKEND,
+});
+
+const authService = new HttpLink({
+  uri: AUTH_FORCE_DEV ? AUTH_DEV : AUTH,
+});
 
 const client = new ApolloClient({
   cache: new InMemoryCache(),
-  uri: BACKEND,
+  link: ApolloLink.split(
```

Comment:
@AjayDoddapaneni This is the change using which we can fire queries to different API endpoints for GraphQL.